### ISSUE TITLE
the target is redis cluster with auth, setup failed 

### ISSUE THREAD

**Issue by andyli029** [vipshop/redis-migrate-tool#3]
**#./src/redis-migrate-tool -c rmt.conf** 
```
[2016-09-13 15:34:29.865] rmt_core.c:525 Nodes count of source group : 1
[2016-09-13 15:34:29.865] rmt_core.c:526 Total threads count : 24
[2016-09-13 15:34:29.865] rmt_core.c:527 Read threads count assigned: 1
[2016-09-13 15:34:29.865] rmt_core.c:528 Write threads count assigned: 1
[2016-09-13 15:34:29.867] rmt_redis.c:6702 ERROR: read from 10.11.144.111:6771 command CLUSTER NODES
 failed: Operation now in progress
[2016-09-13 15:34:29.867] rmt_redis.c:6951 ERROR: no valid server address in the target redis cluster
[2016-09-13 15:34:29.867] rmt_redis.c:579 ERROR: Redis cluster init failed
[2016-09-13 15:34:29.867] rmt_core.c:2291 ERROR: Target redis group init from conf file failed
[2016-09-13 15:34:29.867] rmt_core.c:167 ERROR: Target group create failed
[2016-09-13 15:34:29.867] rmt_core.c:2408 Error: assign threads failed.
```
the reason is that before the operation
```
    if (rmt_sync_write(tc->sd,REDIS_COMMAND_CLUSTER_NODES,
        rmt_strlen(REDIS_COMMAND_CLUSTER_NODES),1000) == -1){
        log_error("ERROR: send to %s command %s failed", 
            node->addr, REDIS_COMMAND_CLUSTER_NODES);
        goto error;
    }
```
sending the auth.

**Comment by andyli029** [comment]
the issue is fixed： #4

**Comment by deep011** [comment]
Good job！
